src/Dapr.Actors/Runtime/TimerInfo.cs

```diff
@@ -24,7 +24,6 @@ namespace Dapr.Actors.Runtime
     /// <summary>
     /// Represents the details of the timer set on an Actor.
     /// </summary>
-    [Obsolete("This class is an implementation detail of the framework and will be made internal in a future release.")]
```

Comment:
Did we make this internal? I don't think we did. And if we did, that's a breaking change so we shouldn't just do it. Any reason why you removed this?